### DISCUSSION TITLE
Up and running

### DISCUSSION THREAD

**Discussion by barrettepp** [aaronwmorris/indi-allsky#1137] [General]
I can’t post to the allsky facebook group all my posts sit, needing to be approved by admin.   I wanted to give an update.  I have my allsky camera together and running.  I have a few tweaks to make, NVMe is not working.  Raspberry pi mounting plate is too wide when plugged in. Need to mount fan.   Then the hard part of focusing camera and getting settings right.  Any tips would be appreciated.    
![IMG_5763](https://github.com/aaronwmorris/indi-allsky/assets/156556984/d2f3e565-3aeb-4db4-8375-a725b26348ce)
![IMG_5762](https://github.com/aaronwmorris/indi-allsky/assets/156556984/bd227b4f-ce0b-4989-a011-765b5c0b317e)
![IMG_5746](https://github.com/aaronwmorris/indi-allsky/assets/156556984/017cc3bd-8cca-4b3c-b03f-264c6966e765)
o

**Comment by aaronwmorris** [comment]
Looking good.

I feel your pain about focusing.  It is one of the most difficult things to achieve.  There is a Focus Mode in indi-allsky where it can take exposures more rapidly and somewhat of a "score" to help.  You have to login with a user to see the tool.  It has a bit of a learning curve.

Personally, I have found that using video streaming is the easiest way to focus.  I generally focus with the camera completely removed from the enclosure.

Firecapture is my preferred tool for focusing.  It is really easy to set something like a 400x400 region of interest [RoI] near the center of the image and adjust focus with instant feedback.  During the day, it is best to try to focus on something in the sky, like a plane, or even the moon if it is above the horizon.  At night, stars make the best target with sufficiently high gain.  I would be careful trying to focus using clouds, it can work, but they can trick you at times.

You can also stream video with INDI and EKOS, but setting a RoI and framerate is a little more difficult.

Use the set screw if you have one on your lens, sometimes you have to be careful as the screw can cause focus shift.  The screw does not need to be locked down, just finger tightened is fine.
